`CalcJob`: Extend `retrieve_list` syntax with `depth=None` (#5651)
aiidateam/aiida-core: docs/source/topics/calculations/usage.rst

If the exact file or folder name is not known beforehand, glob patterns can be used.
In the following examples, all files that match ``*c.txt`` in the directory ``path/sub`` will be retrieved.

> If the exact folder hierarchy should be maintained, the ``depth`` should be disabled by setting it ``None``:

Reviewer: ```suggestion
In order to maintain the folder structure, set ``depth`` to ``None``:
```
